src/graphql/utils/withCache.ts

```diff
@@ -1,22 +0,0 @@
-import memoryCache from 'memory-cache'
```

Comment:
We don't want to cache anymore? Or do we have a different cache? If we are removing this we can also get rid of the `memory-cache` and `@types/memory-cache` dependencies.

Reply:
Cache is now handled by the dataloader and LRUMap for all scenarios

Reply:
Ok, perfect :ok_hand: